analytics/analytics.go

```diff
@@ -48,13 +50,15 @@ type BuildAnalytics struct {
 
 // StepAnalytics ...
 type StepAnalytics struct {
-	StepID      string        `json:"step_id"`
-	StepTitle   string        `json:"step_title"`
-	StepVersion string        `json:"step_verion"`
-	StepSource  string        `json:"step_source"`
-	Status      string        `json:"status"`
-	Runtime     time.Duration `json:"run_time"`
-	StartTime   time.Time     `json:"start_time"`
+	StepID             string        `json:"step_id"`
+	StepTitle          string        `json:"step_title"`
+	StepVersion        string        `json:"step_verion"`
+	StepSource         string        `json:"step_source"`
+	Status             string        `json:"status"`
+	Runtime            time.Duration `json:"run_time"`
+	StartTime          time.Time     `json:"start_time"`
+	SimulatorDevice    string        `json:"simulator_device"`
```

Comment:
this should be a bit more general property, like map[string]string, this way later we can easily add more inputs to submit